monero-outreach-docs/translations/es/archive/we_accept_monero_es.md

```diff
@@ -0,0 +1,19 @@
+# Aceptamos Monero
+
+**“¿Estás cansado de implorar por adopción? ¿Harto de esperar el amanecer de Monero? Se parte del cambio que quieres ver en el ecosistema.” - 20th of MAy, 2020**
+
+Nos gustaría invitarte a unirte a nuestros esfuerzos para llevar a cabo una buena 'campaña agobiante' a la antigua con tus comerciantes favoritos que aún no aceptan XMR. La 'campaña agobiante' más impresionante y exitosa de todos los tiempos tuvo que haber sido escrita por George Lois. En 1982, las empresas de cable no tenían interés en un canal de música las 24 horas, no creían que funcionaría. A George Lois se le ocurrió una idea y filmó una serie de comerciales en los que las estrellas de rock pedían a los espectadores que llamaran a su compañía de cable local y dijeran "¡Quiero mi MTV!" El resto es historia, [youtu.be/vAyOcWpm3QU](https://youtu.be/vAyOcWpm3QU?t=71).
+
+La misión, si decides aceptarlo, es comunicarte con tu comerciante favorito y preguntarle si ha considerado aceptar Monero, porque ese sería su método de pago preferido. Puede que digan que sí, que digan que no, pero al menos les has hecho pensar en ello. No será la última vez que escuchen sobre Monero, pero podría ser la primera.
+
+En marketing se dice que alguien necesita ver un mensaje siete veces antes de darse cuenta de él. Esa es una regla general para la publicidad, y esperamos fomentar una conversación con los comerciantes. Una buena conversación vale fácilmente siete anuncios publicitarios. La investigación ha encontrado que las personas que conocen una organización tienen cinco veces más probabilidades de tener una opinión favorable de ella. En otras palabras, cuanto más hables de Monero, a más gente le gustará.
+
+También hemos creado este folleto del tamaño de un billete de un dólar que puedes imprimir y guardar en tu billetera cuando estés en la calle. Este artículo se centra en las tres características únicas de Monero que son las más importantes para los propietarios de pequeñas empresas y los invita a aprender más en nuestras páginas de [Monero Merchant Services] (https://www.monerooutreach.org/merchants/).
```

Comment:
Please remove the space between the square brackets [] and the normal brackets ().

[Monero Merchant Services] (https://www.monerooutreach.org/merchants/)

[Monero Merchant Services](https://www.monerooutreach.org/merchants/)

Reply:
Done!

Reply:
The space was removed